run.sh

```diff
@@ -0,0 +1,2 @@
+cd hotel-api-server
```

Comment:
Sorry, what is this file for?

Reply:
I was just trying to create a shortcut to run the app. Only for personal fast execution. Is it correct to have a file like this in the codebase?